src/lib/middleware/api-token-middleware.ts

```diff
@@ -7,7 +7,16 @@ const isClientApi = ({ path }) => {
 };
 
 const isProxyApi = ({ path }) => {
-    return path && path.startsWith('/api/frontend');
+    if (!path) {
+        return;
+    }
+
+    return (
+        (!path.startsWith('/api/client') &&
+            !path.startsWith('/api/admin') &&
+            path.includes('proxy')) ||
```

Comment:
Seems like `path.includes('proxy')` may match more than we want. Could we be stricter?